[resource][vnet] add AR make crash

Provider: 0.4.0

There is a crash when adding an AR to a virtual network:
```
resource "opennebula_virtual_network" "network1" {
  name   = "test-net1"
  type   = "bridge"
  bridge = "onebr"
  mtu    = 1500
  ar {
    ar_type = "IP4"
    size    = 12
    ip4     = "172.16.100.130"
  }
  permissions     = "642"
  group           = "oneadmin"
  security_groups = [0]
  clusters        = [0]
}
```
Then add and apply again:
```
...  
ar {
    ar_type = "IP4"
    size    = 12
    ip4     = "172.16.100.145"
  }
...
```

Result:
```
Stack trace from the terraform-provider-opennebula plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 57 [running]:
github.com/OpenNebula/terraform-provider-opennebula/opennebula.resourceOpennebulaVirtualNetworkUpdate(0xc0005670a0, 0xea6560, 0xc000562620, 0x24, 0x1551440)
        /home/pierre/go/src/terraform-provider-opennebula/opennebula/resource_opennebula_virtual_network.go:1089 +0x27ec
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc0001f2900, 0xc0003c6370, 0xc0006634e0, 0xea6560, 0xc000562620, 0xda4d01, 0xc0001ab928, 0xc000396840)
        /home/pierre/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.7.0/helper/schema/resource.go:311 +0x273
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc0001f2c00, 0xc0002938e8, 0xc0003c6370, 0xc0006634e0, 0xc000118198, 0xc00042c001, 0xda69e0)
        /home/pierre/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.7.0/helper/schema/provider.go:294 +0x99
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc0001b8a28, 0x102afb0, 0xc000500e70, 0xc0005c81e0, 0xc0001b8a28, 0xc000500e70, 0xc000077a50)
        /home/pierre/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.7.0/internal/helper/plugin/grpc_provider.go:885 +0x8a5
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0xe7a3c0, 0xc0001b8a28, 0x102afb0, 0xc000500e70, 0xc0005c8180, 0x0, 0x102afb0, 0xc000500e70, 0xc0003cb800, 0x798)
        /home/pierre/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.7.0/internal/tfplugin5/tfplugin5.pb.go:3189 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001c8000, 0x10325f8, 0xc000001b00, 0xc00009a200, 0xc0006964e0, 0x15126e0, 0x0, 0x0, 0x0)
        /home/pierre/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:995 +0x482
google.golang.org/grpc.(*Server).handleStream(0xc0001c8000, 0x10325f8, 0xc000001b00, 0xc00009a200, 0x0)
        /home/pierre/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:1275 +0xd2c
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000422270, 0xc0001c8000, 0x10325f8, 0xc000001b00, 0xc00009a200)
        /home/pierre/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:710 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /home/pierre/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:708 +0xa5

Error: The terraform-provider-opennebula plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

Introduced by #141

--- a/opennebula/resource_opennebula_virtual_network.go
+++ b/opennebula/resource_opennebula_virtual_network.go
@@ -1075,23 +1075,6 @@ func resourceOpennebulaVirtualNetworkUpdate(d *schema.ResourceData, meta interfa
 			"prefix_length",
 		)
 
-		// reorder arToAdd list according to new list order
-		newARToAdd := make([]interface{}, len(ARToAdd))
-
-		i := 0
-		for _, newARIf := range newARsCfg {
-			newAR := newARIf.(map[string]interface{})
-
-			for _, ARToAddIf := range ARToAdd {
-				AR := ARToAddIf.(map[string]interface{})
-
-				if (AR["ip4"] == newAR["ip4"] || AR["ip6"] == newAR["ip6"]) && AR["size"] == newAR["size"] {
-					newARToAdd[i] = AR
-					i++
-				}
-			}
-		}
-
 		// remove ARs
 		for _, ARIf := range ARToRem {
 			ARConfig := ARIf.(map[string]interface{})
@@ -1108,7 +1091,7 @@ func resourceOpennebulaVirtualNetworkUpdate(d *schema.ResourceData, meta interfa
 		}
 
 		// Add new ARs
-		for _, ARIf := range newARToAdd {
+		for _, ARIf := range ARToAdd {
 			ARConfig := ARIf.(map[string]interface{})
 
 			ARTemplateStr := generateAR(ARConfig).String()

